ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -2609,7 +2610,11 @@ impl<'a> Assemble<'a> {
                             );
                         }
                         VarLocation::ConstInt { bits, v } => {
-                            self.ra.assign_const(iidx, bits, v);
+                            self.ra.assign_const_int(iidx, bits, v);
+                        }
+                        VarLocation::ConstPtr(v) => {
+                            // unwrap can't fail because pointers are sized.
```

Comment:
There is no `unwrap` here.

Reply:
Fixed in f9da2df36725f8e28714bb64acd9e52604b9c73f